src/main/java/cartPage/ApplyCouponCodeTable.java

```diff
@@ -0,0 +1,10 @@
+package cartPage;
+
+import org.openqa.selenium.By;
+
+public class ApplyCouponCodeTable {
+
+    public final String applyCouponCodeTable = "//*[@id='cart']/div/div[2]/div[1]";
+    public By couponLocator = By.xpath(applyCouponCodeTable + "//input[@id='coupon_coupon']");
```

Comment:
It will be good to have some validation methods here?:)

Reply:
The same as above. I will add some methods there